README.md

```diff
@@ -12,7 +12,8 @@ Tools for working with tabular data in Julia.
 
 **Installation**: at the Julia REPL, `Pkg.add("DataFrames")`
 
-**Documentation**: [available here](http://dataframesjl.readthedocs.org/en/latest/)
+**Stable Documentation**: [available here](http://JuliaStats.github.io/DataFrames.jl/stable/)
+**In-Development Documentation**: [available here](http://JuliaStats.github.io/DataFrames.jl/latest/)
```

Comment:
Said that way, one might think the documentation is stable/in development. Maybe say "Documentation for stable version"?